Fix #518 for 03-create

gcapes opened the pull request:
Add solutions, provide working directory context, and where necessary modify the exercises to work with files in data-shell (so that learners can test commands rather than having to work everything out in their heads).

shwina commented on _episodes/03-create.md:
```diff
@@ -556,43 +556,54 @@ but it does find the copy in `thesis` that we didn't delete.
 > {: .output}
 >
 > > ## Solution
```
Why get rid of the text explanation?

gcapes replied:
I think it should stay after all. Perhaps I was going to add it to the exercise text instead.

shwina commented on _episodes/03-create.md:
```diff
@@ -556,43 +556,54 @@ but it does find the copy in `thesis` that we didn't delete.
 > ## Copy with Multiple Filenames
 >
-> What does `cp` do when given several filenames and a directory name, as in:
+> For this exercise, you can test the commands in the data-shell/data directory.
```
Use backticks: `data-shell/data`

shwina commented on _episodes/03-create.md:
```diff
@@ -556,43 +556,54 @@ but it does find the copy in `thesis` that we didn't delete.
+> In the example below, what does `cp` do when given several filenames and a directory name?
 >
 > ~~~
+> $ cd ~/data-shell/data
```
I would leave out the `cd` here. Many participants will have put the `data-shell` directory in some other location (not `~`).

shwina commented on _episodes/03-create.md:
```diff
@@ -556,43 +556,54 @@ but it does find the copy in `thesis` that we didn't delete.
 > > ## Solution
-> > When given several filenames and a directory name, `cp` copies the files to the destination directory (the last argument).
-> > When given three or more filenames, `cp` expects the last argument to be the destination directory, therefore it returns an error: `target survey.txt is not a directory`.
+> > If given more than one file name followed by a directory name, `cp` copies the files
```
Let's make it explicit that the directory name must be the last argument

gcapes commented:
Thanks for the review. Do I just rebase onto the same branch, after fixing the points in the comments?

gcapes commented:
Hi @shwina, 

I hope this is what you intended? I've rebased and fixed the issues addressed in your comments. I had to force push the rebased branch, which means the original commits have disappeared/now show as out of date. 

I also dropped the commit about `Ctrl-R` because that has already been merged and shouldn't have been on this branch. 

Could you confirm this is how I should do the rebasing?
Thanks

shwina commented:
Yup - it looks good to me. If you are done adding commits, I will merge.

gcapes commented:
Excellent! All done. I'll let you get back to reviewing the other PRs then ;)
